src/ansys/motorcad/core/rpc_client_core.py

```diff
@@ -14,7 +14,9 @@
 
 DEFAULT_INSTANCE = -1
 
-SERVER_IP = "http://localhost"
+LOCALHOST_ADDRESS = "http://localhost"
+TRY_RESOLVE_LOCALHOST = True
```

Comment:
Flag so we can enable/disable this behaviour. Potentially useful for testing/disabling if there are issues